src/themes/playful.css

```
@@ -244,22 +260,22 @@
   /* Border widths are designed to scale according to a single base value.
      * The base value is intended for calculations and is not used by components directly. */
   --wa-border-width-base: 0.125;
-  --wa-border-width-s: calc(var(--wa-border-width-base) * 1rem);
+  --wa-border-width-s: calc(var(--wa-border-width-base) * 1.5rem);
   --wa-border-width-m: calc(var(--wa-border-width-base) * 2rem);
   --wa-border-width-l: calc(var(--wa-border-width-base) * 3rem);
 
   --wa-form-controls-border-style: var(--wa-border-style);
-  --wa-form-controls-border-width: var(--wa-border-width-s);
+  --wa-form-controls-border-width: var(--wa-border-width-m);
 
   --wa-panel-border-style: var(--wa-border-style);
-  --wa-panel-border-width: var(--wa-border-width-s);
+  --wa-panel-border-width: 0;
```

```suggestion
  --wa-panel-border-width: 0rem;
```
Adding a unit here lets us account for border widths in calculations later on.